### PR TITLE
Fix compatibility with tensorflow 1.14.0

### PR DESCRIPTION
**Pull request by yorickvP**
This blacklists all the ops that use 'func'. Long term it might be better to support it.

### CODE REVIEW

**Comment by yorickvP** [comment]
I signed it!

**Comment by blackgnezdo** [comment]
Could you update a few more files where we have the version baked-in? Unfortunately it is duplicated:
https://github.com/tensorflow/haskell/commit/ecfea97992c0a8dd0f2385e96658fef7e5ca23bf

**Comment by yorickvP** [comment]
I think that's all of them!

**Comment by yorickvP** [comment]
I can't see why the build fails, the details page is on an internal network.

**Comment by blackgnezdo** [comment]
I guess they pruned the image. Could you add curl?
```
Processing triggers for libc-bin (2.23-0ubuntu11) ...
/bin/sh: 1: curl: not found
The command '/bin/sh -c apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 575159689BEFB442 &&     apt-get update &&     apt-get install -y         libsnappy-dev         libncurses5-dev         libgmp-dev         libcurl3-dev         netbase         &&     curl -O -L https://github.com/commercialhaskell/stack/releases/download/v1.7.1/stack-1.7.1-linux-x86_64.tar.gz &&     tar zxf stack-1.7.1-linux-x86_64.tar.gz -C /usr/local/bin stack-1.7.1-linux-x86_64/stack --strip 1 &&     curl -O -L https://github.com/google/protobuf/releases/download/v3.2.0/protoc-3.2.0-linux-x86_64.zip &&     unzip -d /usr/local protoc-3.2.0-linux-x86_64.zip bin/protoc &&     chmod 755 /usr/local/bin/protoc &&     curl -O https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-cpu-linux-x86_64-1.13.1.tar.gz &&     tar zxf libtensorflow-cpu-linux-x86_64-1.13.1.tar.gz -C /usr/local &&     ldconfig &&     stack setup &&     stack test --only-dependencies' returned a non-zero code: 127
[ID: 7243153] Build finished after 80 secs, exit value: 127
```

**Comment by yorickvP** [comment]
Curl has been added. Since 1.14 is out, maybe I'll update everything to that.

File: README.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ stack --docker --docker-image=$IMAGE_NAME build --exec Main
 If you want to use GPU you can do:
 
 ```
-IMAGE_NAME=tensorflow/haskell:1.9.0-gpu
+IMAGE_NAME=tensorflow/haskell:1.4.0-gpu
```
Comment (blackgnezdo) [inline README.md]: This looks like a downgrade. Undo?

Comment (yorickvP) [reply README.md]: Yes, that's a typo. Thanks for noticing!

**Comment by vilunov** [comment]
@blackgnezdo Hi! Sorry to bother, but are there any problems with this PR? CI seems to be stuck

**Comment by blackgnezdo** [comment]
Apologies, if the build goes through, @fkm3 or @judah could you merge this?

**Comment by fkm3** [comment]
I guess they really pruned down the image. The current error is

```
/bin/sh: 1: unzip: not found
```

If you've got a docker setup locally, you should be able to run the follow command to reproduce the error.

```
haskell/ci_build/outer_launch_tests.sh
```

**Comment by vilunov** [comment]
Another dependency you need is `zlib1g-dev`, without it the stack build fails

**Comment by fkm3** [comment]
Yeah. I just got a successful run of the CI tests locally by adding both `unzip` and `zlib1g-dev` to the `Dockerfile`s.

**Comment by blackgnezdo** [comment]
@fkm3 could you check what's broken now?

**Comment by vilunov** [comment]
@yorickcP seems you forgot a `\` symbol at the end of the line in some of
dockerfiles

**Comment by blackgnezdo** [comment]
> @yorickvP seems you forgot a `\` symbol at the end of the line in some of dockerfiles

Thanks @vilunov, let's see if I can make it go through.

**Pull request merged**